### ISSUE TITLE
Remove Three Tiers

### ISSUE THREAD

**Issue by loganfranken** [UCSantaBarbara/webguide#1]
When the Web Standards Guide Rewrite Subcommittee reviewed the web guide a couple years ago, we kept the "Three Tiers" page (http://webguide.ucsb.edu/tiers/), although we removed all links and references to it beyond a link in the "About" section (http://webguide.ucsb.edu/about/).

As I remember, the motivation behind this was that the best practices outlined in the web guide are applicable to _every_ UCSB website, so every website should strive to follow them. The page was still retained, however, for historical reference.

I would suggest **removing the page completely**: otherwise we are featuring and implicitly supporting deprecated content on the web guide.

**Comment by tenken** [comment]
I say remove it. -dg

**Comment by loganfranken** [comment]
URL of the page on the GitHub-hosted guide:
http://ucsb-wsg.github.io/tiers/

@tenken I don't mind making this update, but I imagine we may want to discuss this within the group first?

**Comment by loganfranken** [comment]
During the meeting today, the group didn't agree on removing this page. Instead, the group opted to flesh out the "Examples Page" (#34) first to ensure that it will serve as a sufficient replacement

**Comment by tenken** [comment]
So to be clear, should I reject the pending merge request for the meeting details to be fleshed out in a new branch?

**Comment by UCEAP-JCOLON** [comment]
http://ucsb-wsg.github.io/tiers/ will no longer be its own page. It will be on a forth-coming examples page. 
See issue: https://github.com/ucsb-wsg/ucsb-wsg.github.io/issues/34
Thanks!

**Comment by loganfranken** [comment]
@tenken It's totally up to you: you can just reject it and I'll try again after #34 is resolved or we can leave it out and see if it can be cleanly merged later.

**Comment by loganfranken** [comment]
Alright, now that we have the "Examples" page in place, we can revisit removing this page.

Would it be possible to review and vote on this at next month's meeting?

**Comment by UCEAP-JCOLON** [comment]
Cool,
I certainly think so.
thanks Logan!

**Comment by loganfranken** [comment]
Could we add the label to mark this as something to discuss at the next WSG meeting?

**Comment by loganfranken** [comment]
Hey @UCEAP-JCOLON or @tenken; could we pop an "Item for WSG Meeting" label on this one?

**Comment by UCEAP-JCOLON** [comment]
My understanding from the last WSG meeting was that we were to move the tiers to the examples page so we can remove the stand alone page.. Do we need to revisit this at the WSG meeting? I think we were on pace to remove having the content on the examples page..

**Comment by loganfranken** [comment]
Hey @UCEAP-JCOLON!

My understanding is that we are _removing_ the tiers concept altogether: basically, some of the group members were uncomfortable with removing the "Three Tiers" page if we didn't have an "Examples" page with content from the old Tiers page.

Now that we have an "Examples" page in place (http://ucsb-wsg.github.io/examples/), I think the next step is to get consensus from the group to actually remove the Tiers page.

Let me know if I misunderstood anything!

**Comment by UCEAP-JCOLON** [comment]
Hey Logan, thats right, its more about the examples than the tiers concept. I think then we can discuss at the next meeting. thanks ~James

**Comment by loganfranken** [comment]
Thanks @UCEAP-JCOLON! Would you mind popping on the label so we remember?

**Comment by UCEAP-JCOLON** [comment]
All set!
Thanks

**Comment by loganfranken** [comment]
Hey @UCEAP-JCOLON! Sorry, to pester: but I still don't see the "Item for WSG Meeting" label.

**Comment by UCEAP-JCOLON** [comment]
Sometime it takes two clicks to save. :)
I see it showing now.
thanks

**Comment by loganfranken** [comment]
I see it now! Thanks @UCEAP-JCOLON!

**Comment by loganfranken** [comment]
My connection dropped out when we finished discussing this: are we okay with removing the Three Tiers now?

I thought the contention was that there was a link between the Three Tiers and a site's adherence to the Graphic Identity which I think is no longer true since Public Affairs will be taking over guidance for branding/visual presence.

Or maybe I got that all twisted?

**Comment by tenken** [comment]
You can see the brief note I put about it in today's meeting minutes:
https://it.ucsb.edu/groups/wsg/wsg-july-2015-meeting

Basically the group would like some verbiage to remain saying the Identity
of the site should reflect how close or core a service the site is with
UCSB. This is essentially for the intermim while PA drafts up their website.

On a related note lately I've been looking at tools like gitbook, (php)
couscous, Read The Docs (python) -- for simple documentation based sites to
be released internally with projects for clients. Sure PA could use github
(and maybe gitbook) to release a branding guidelines site -- but I would
recommend 1 of these tools as a simple means of creating a flat-file CMS
that is living and more "book-like" (and themeable) than say straight
ghpages. Is a github wiki themable!?

**Comment by UCEAP-JCOLON** [comment]
It sounds like several folks want the three tiers area to remain as a sample of branding versus accessibility. Even though Public Affairs will be giving guidance it seems some folks with lower tiers sites still want flexibility to deviate from the path of the main sites branding (ucsb.edu), Thus they want the tiers page to remain to communicate at what level they can be less rigid about branding consistency...

**Comment by loganfranken** [comment]
Ahh, gotcha.

To be clear, there's been a lot of talk about some kind of connection between the Three Tiers and visual identity.

There may have been some kind of historical connection/context there, but at least as far as I know, there has been no explicit statement making that connection.

From the [Three Tiers page](http://webguide.ucsb.edu/tiers/) that was ported over from the legacy Guide:

> A website's position reflects how strongly we believe that site should adhere to the standards defined in this guide.

So if you're Tier 1, the implication here is that we _strongly_ encourage you to adhere to the standards. Arguably, this includes the standards on the Graphic Identity page.

To that end, here are the actual "standards" listed on the Graphic Identity page:

> Identify UC Santa Barbara and your department to make it clear that your site is part of UCSB, and what department or unit the site belongs to. We recommend that the University's name or logo and the department name be displayed clearly above the fold.
> 
> Websites using [UCSB copyrighted images] should include the copyright information in the footer.
> 
> By downloading images from this site you are agreeing to follow the standards set forth in [the Terms of Use, Use of University's Name, Use of University's Seal]
> 
> We encourage you to choose your blue and gold from this palette
> 
> For body copy, use a sans-serif font widely supported by browsers, such as Arial or Verdana.
> 
> When specifying fonts in your CSS, don't assume that any particular font is available. Offer a generic alternative style ... in your font stack

This is all framed by a disclaimer at the top of the page:

> We are issuing no dictates about the "look and feel" of your websites, and simply wish to provide this page as a resource.

All of the standards above are stuff that seems to be broadly applicable (although I would be totally open to arguments otherwise) except for the following:

> We encourage you to choose your blue and gold from this palette

However, these colors are not reflected in a majority of websites: most websites just play off this color scheme.

Given all of this, I don't really see the association between the Tiers and a site's graphic identity. I'm assuming it must have existed at some point? But it hasn't existed in quite some time as far as I know.

If anyone's wondering, the core text of these pages has been more or less the same since 2007:
https://web.archive.org/web/20070112010432/http://www.ucsb.edu/webguide/tiers.shtml
https://web.archive.org/web/20070112010538/http://www.ucsb.edu/webguide/graphic-identity.shtml

**Comment by fablio-work** [comment]
My take away was that the three tiers should now be focused on the visual identity aspect of the standards.  
So possibly some adjustment to the verbage on that page is in order.
I know that I personally was originally thinking of the three tiers more in terms of accessibility policies rather 
than visual identity, in which case it makes sense that changes in policy had invalidated the page.  
But, several folks at the meeting mentioned that the tiers are a useful tool in helping
prevent a supervisor from "going rogue” on a site design.

Aaron Martin  (CNT)
Earth Research Institute
6809 Ellison Hall
UCSB MS-3060
805-893-8415
aaron@eri.ucsb.edu mailto:aaron@eri.ucsb.edu

**Comment by loganfranken** [comment]
That makes sense, @fablio-work; my only concern is that we don't actually have any real visual identity recommendations in the Web Guide except the few mentioned above (and copied below), which I would argue apply to _all_ UCSB sites.

So this would bring up the same dilemma where, if we say something like the following:

> Tier 1 sites are strongly encouraged to follow the visual identity recommendations on the Graphic Identity page

This implies that non-Tier 1 sites _shouldn't_ follow the recommendations on the Graphic Identity page, which means non-Tier 1 sites _shouldn't_ feel compelled to follow these recommendations:

> Websites using [UCSB copyrighted images] should include the copyright information in the footer.
> 
> By downloading images from this site you are agreeing to follow the standards set forth in [the Terms of Use, Use of University's Name, Use of University's Seal]
> 
> We encourage you to choose your blue and gold from this palette
> 
> For body copy, use a sans-serif font widely supported by browsers, such as Arial or Verdana.
> 
> When specifying fonts in your CSS, don't assume that any particular font is available. Offer a generic alternative style ... in your font stack

But I think all of those apply to _all_ UCSB sites (unless anyone disagrees?), which means a distinction between Tiers in regards to visual identity doesn't make sense.

A possible solution is to create _more_ recommendations surrounding graphic/visual identity so that the Tier distinction has relevance, but this dips into an area that Public Affairs is already going to cover, so I don't think it makes sense for us to jump into that arena.

If people have concerns over site designs "going rogue," they will still be able to rely on our general recommendations above, just like always. This just closes a loop where non-Tier 1 sites are given justification to ignore the broadly applicable (again, IMO) recommendations above.

**Comment by tenken** [comment]
I agree we shouldnt try to get involved in the topic of how close sites appear to keep a visual identity with the campus.

There are always 2 camps to the issue. Those that believe UCSB identity should be prevalent on _any_ site and those campus clients that want a custom snowflake website for every project (every snowflake is 100% unique).

I don't know PA long term goals on the matter -- but many designers and all of my clients to date really dislike homogeneous sites built around a single look and feel. As an example both Stanford.edu and Harvard has some Drupal themes and more broadly an accepted visual identity -- sites built within their infrastructure historically all look extremely similar (but they're accessible and visually consistent).
- Students and staff have a hard time differentiating what site they're on, everything looks the same.
- simply put there is no uniqueness or feedback that "this is mine", when everything is boilerplate or homogeneous.

This may be more prevalent in the past when their frameworks were in their infancy but see Stanfords Jumpstart Sites which shows some example templates, or the OpenScholar "top sites" on the bottom left of their page: 
- https://sites-jumpstart.stanford.edu/
- http://openscholar.harvard.edu/   (this is a really nice dashboard page :D)

**Comment by alexparraga** [comment]
My opinion is to remove the three tiers page. We shouldn't set different expectations for different sites. I also think we should maintain a  strong urge/recommendation for all UCSB sites to adhere to the WSG accessibility standards. That way it's still there for folks who need to lean on it to help gain alignment within their organization. 

@tenken, on the topic of "homogenous sites", I don't think the arguments you've been given are strong ones. For one, differentiating what site you're on is as simple as looking at the URL or the page header. We need to give our visitors more credit than that. And if the site fails to clearly communicate what it's representing, then that's on the site owners/managers. Second, in terms of uniqueness, that's also on site owners/managers. The differentiation needs to be in the messages and content. A site with a strong message and content will be easily differentiated from others, regardless of whether it follows a template or common theme. 

Going forward, we do plan to provide site solutions that help convey a more consistent brand identity across UCSB web properties while allowing for flexibility to differentiate your own dept/organization. In the interim, I'm happy to participate in conversations to help support the benefits of consistency in both UX and look & feel.

**Comment by ghost** [comment]
Hi All,

I participated quite a bit on this issue many years ago...  The tiers are really about flexibility for departments, research labs etc to be free to create their own graphic identity which it is important to these groups.

**Comment by fablio-work** [comment]
> On Jul 9, 2015, at 10:47 AM, alexparraga notifications@github.com wrote:
> 
> My opinion is to remove the three tiers page. We shouldn't set different expectations for different sites. I also think we should maintain a strong urge/recommendation for all UCSB sites to adhere to the WSG accessibility standards. That way it's still there for folks who need to lean on it to help gain alignment within their organization.
> 
> @tenken https://github.com/tenken, on the topic of "homogenous sites", I don't think the arguments you've been given are strong ones. For one, differentiating what site you're on is as simple as looking at the URL or the page header. We need to given our visitors more credit than that. And if the site fails to
> 
> Every user is different and “key” off different things.  I encounter users pretty regularly who do NOT notice differences in the url or subtle changes in appearance or text.  And it’s not just older users.  I have seen a decent number of young adults who just flit from one site to another without really realizing what they are doing and where they are going.  They are looking for specific information and may not be reading the heading or url for every page click.  If the site changes are too subtle, they can be easily missed.

Having said that.  I am still not sure which way this should go.  Logan makes an excellent point about choices in the webguide “giving permission” to not follow whole portions of its recommendations.

**Comment by garster** [comment]
It seems to me here is a strong want to keep it. But I suggest changing the focus to be branding and not web standards. And it needs to be reviewed and kept up to date as new sites come online or go offline.

**Comment by loganfranken** [comment]
Hey everyone! Thank you all _so much_ for participating in this discussion! It's so wonderful to see such a vibrant discussion regarding UCSB's web presence!

To summarize, the main argument I've seen for _keeping_ the Three Tiers is the following:

> Websites that are in the lower tiers should be allowed to develop their own visual identity that may not necessarily align with the official UCSB visual identity

(Let me know if I got that wrong!)

I'm not arguing against the above statement at all. However, my main issue is that this is _not_ accurately reflected in the current Tiers page (as @garster mentioned) nor has it ever been reflected since 2007.

Here is the text from the Tiers page:

> The Web Standards Group has grouped UCSB websites into three tiers, based on various factors: association with the University identity, sphere of influence, frequency and volume of use, expected lifetime, and other factors. **A website's position reflects how strongly we believe that site should adhere to the standards defined in this guide.** All campus units, regardless of the tier, should make every effort to comply with WSG accessibility recommendations. The tiers are described below, followed by examples of selected sites in each tier. Other sites will be added in the future.

It doesn't explicitly call out visual identity; it says "standards defined in this guide", implying _all_ of the standards in this guide. This has the implication that sites in lower Tiers should not follow the recommendations regarding HTML best practices, security, accessibility, and so on, which is a dangerous assertion.

So the follow-up argument could be that we should rewrite it to say "**visual identity** standards defined in this guide."

My issue with this argument is that there aren't really any visual identity standards defined in this guide. I copied them above, but the standards outlined on our [graphic identity](http://webguide.ucsb.edu/ucsb-graphic-identity/) page are as follows:
- Make it clear the website is a UCSB website
- Make it clear which department/unit the site belongs to
- Putting the name "UCSB" or the logo somewhere at the top is _recommended_
- If you use copyrighted images, put copyright information in the footer
- If you download images of the seal, you agree to the standards/policies related to using the seal
- Using a blue and gold color from the specified palette is _encouraged_
- Use a widely supported sans-serif font for body copy
- Offer a generic alternative in your font stack

(Let me know if I missed something!)

None of these seem like wild demands to me: they are all pretty soft ball recommendations IMO.

Even if I'm wrong and a particular department disagrees with these standards, there's a caveat at the very top of the [graphic identity](http://webguide.ucsb.edu/ucsb-graphic-identity/) page:

> We are issuing no dictates about the "look and feel" of your websites, and simply wish to provide this page as a resource.

So to make this whole Tiers page work as outlined, we would have to:
- Rewrite the Tiers page to specifically call out "visual identity"
- Rewrite the Graphic Identity page to include more direct and specific rules so that there's actually visual identity policy being enforced by the Tiers

And that second bullet point seems to contradict with the fundamental reason for keeping the Tiers page in the first place.

My other major concern is a philosophical one: I personally feel like the Tiers concept introduces a bureaucratic division between sites, labeling some as less "important" than others based on arbitrary criteria.

I really think we should push for having every site, whether it's the Admissions site or a small site created for some departmental project, to feel as if they are all at the same level and all participating in moving UCSB's web presence forward.

**Comment by ghost** [comment]
All sites should try to adhere to 508 / markup standards but with so many tools available today there will be a lot to cover. The bigger issue now are to get people using adaptive frameworks so there sites work on all platforms.

**Comment by tmendez10** [comment]
I am late to this conversation - but my input is that all UCSB websites should work towards compliance with the policies listed on the WSG page and IT Accessibility Policy / section 508. 

In as much as the "tier" system creates confusion that UCSB websites do not have to follow the policies and regulations listed by WSG, then I am in favor of moving away from the tier system. For example, it is a requirement - not a recommendation - that course sites be web accessible if the students are required to access the course site and there is a student with an applicable disability in the class. 

The only distinguishing category I can see is between UCSB sites and non-UCSB sites. I would note that certain non-UCSB sites may have slightly different policy and regulatory requirements than UCSB sites. For example, while student organizations (like fraternities or sports clubs) are permitted to use the UCSB name to indicate their location they generally may not use the UCSB name or logo to indicate the University's endorsement. Depending on how a faculty member's site is configured - it may also be considered a non-UCSB site. (If the site is something that the faculty member is personally developing and hosting then it would likely not be a UCSB site). 

Let me know if you have questions. Tessa Mendez

**Comment by secabeen** [comment]
For me, I think the tiers have value in communicating to developers how responsive to changes in standards various sites have to be.  I think we can all agree that all sites should strive to meet all WSG standards when originally developed.  However, standards change and sites don't.  As I see it, a Tier 1 site is one who's owners should be expected to be following new standards as they are developed, and should be in compliance with new policies and standards when they are finalized, or within a few weeks.  A Tier 2 site is one where the owners may not see the site as their primary responsibility, but can be expected to become compliant with a changed standard or policy within 6-12 months of the standard being adopted.  A Tier 3 site is one where the owners only engage with the site as necessary.  Tier 3 sites are expected to meet the web standards at the time the site is developed or significantly re-worked, but are not expected to be updated to meet new standards or policies adopted after the site is launched.

Does that make sense?

**Comment by loganfranken** [comment]
@secabeen I think that totally makes sense! (And your sentiment aligns with the current text on the Web Guide)

I'm personally uncomfortable with this approach as I think it implies that Tier 3 sites are given an official justification to _not_ strive for best practices. The Guide has never been about enforcement; it's always been about recommendations, encouragement, and support.

To that end, I'm in favor of cultivating a common end goal for all sites, regardless of their perceived importance to the University.

At the end of the day, whether or not people can actually _achieve_ that goal will depend on a number of realistic factors, like available resources, ability to update the site, technical knowledge, and so on.

**Comment by alexparraga** [comment]
Completely agree with @loganfranken. 

With regards to brand standards, we've previously discussed removing/separating the graphic identity content from the web guide given how outdated it is. Additionally, incorporating graphic identity standards is not really in line with WSG's stated mission of helping "campus website developers create POLICY COMPLIANT websites that utilize industry best practices and policies set forth by the federal government and the University of California." I think in the past it was helpful to have them in the absence of anything else, but Public Affairs is now in the process of developing an interim graphic identity page which will live on the ucsb.edu domain and include recommendations on branding elements in line with the direction that institutional channels (such as ucsb.edu) are heading in. That page will be consistent with the style guide that was circulated via the WSG list serve earlier in the week, and will be in place until a more robust brand identity system is developed.

All that said, keeping a tier system for the purposes of branding compliance (or lack thereof) will no longer be as relevant going forward. That, coupled with Logan's point about having a common end goal for all sites, are strong reasons to remove the tier system.

**Comment by BwolfUCSB** [comment]
I disagree, the WSG was formed as a guide for web standards which included branding recommendations.

Chees! B

Brian Wolf
Web Designer / Developer
Life Sciences Computing Group

Phone: (805) 893-5932
Email: brian.wolf@lscg.ucsb.edu
Office: 2302 Life Sciences Building
Website: webdesign.lscg.ucsb.edu

**Comment by ghost** [comment]
I disagree, the WSG was formed as a guide for web standards which included branding recommendations.

Cheers! B

Brian Wolf
Web Designer / Developer
Life Sciences Computing Group

Phone: (805) 893-5932
Email: brian.wolf@lscg.ucsb.edu mailto:brian.wolf@lscg.ucsb.edu
Office: 2302 Life Sciences Building
Website: webdesign.lscg.ucsb.edu http://webdesign.lscg.ucsb.edu/

**Comment by alexparraga** [comment]
That is not how the mission currently reads. 

This issue does not seem like one that can be resolved over a group chat. I recommend scheduling a meeting dedicated to discussing this.

**Comment by ghost** [comment]
Sounds good.

Brian Wolf
Web Designer / Developer
Life Sciences Computing Group

Phone: (805) 893-5932
Email: brian.wolf@lscg.ucsb.edu
Office: 2302 Life Sciences Building
Website: webdesign.lscg.ucsb.edu

**Comment by loganfranken** [comment]
Are we thinking next WSG meeting? /cc @tenken @UCEAP-JCOLON

**Comment by mer-ucsb** [comment]
I have been a member of the WSG group since its inception.

Historically both branding and accessibility drove this WSG Tier definition page. This may have not be stated directly, but that was the historical intent.

Accessibility is the law. From my perspective, on this matter, there should be little/no distinction made between the Tiers (important/highly trafficked sites versus less so). 

The importance of branding has not changed. (New coke versus "classic" code ... need I say more).
The Tier page regarding branding should remain as a guide/resource for web developers (and for the committees involved in designing  UCSB sites). The original intent of the Tier classification was to support UCSB Public Affairs office (Andrew Teton et al), other UCSB Administration offices (eg: Vice Chancellor Sheldon etc.) and a few academic department efforts to get important/highly trafficked UCSB sites to appear as if we are all part of the same corporal body. WSG did not think it user-friendly to require the user to check the url to make sure they are still within UCSB's purview. Public Affairs was trying to get this branding synergy with print media and WSG was helping to extend this with digital media.

This matter should be discussed in a WSG meeting - as it was in June and July's meeting. The WSG mailing list discussion also has pertinent points that need to be included. To state the obvious - not all members of the WSG group have Github accounts.

Margaret Rankin
UCSB Library

**Comment by garster** [comment]
I am fairly new to UCSB, and its "federated" model. I love to learn about why things are the way they are. I have never seen this side of WSG before, only the "cool toys" part, and the talk to peers to see what they are doing part. Since UCSB does not follow a chain of command to force things getting done, groups like WSG must be created to sway others. Very interesting mode of operation, very inefficient, but I guess this is the "government work" I have heard about. 

I have to agree with both Logan and Margaret on different points. I think the compromise is to keep the page but modify the page to highlight graphic identity, then we can all still run back to our managers to point them to back to this to allow us to get things done. 

I do need to point out, I can't even find the page on http://webguide.ucsb.edu/

**Comment by loganfranken** [comment]
Quoted from an e-mail Mark Grosch (UCSB Adaptive Technology Specialist and ADA Web Compliance Officer) sent (with his permission) in a separate e-mail thread that spun off the original thread sent over the UCSB Web Mailing List:

> As far as having two tiers for content layout/styles, I don’t really have a preference either way and I can see both sides of that argument.  However in terms of accessibility, the DOJ has been quite clear as to how they are going to proceed from here on out.  Some background material concerns a few cases brought against private universities for non-captioned videos can be found here:
> 
> http://www.jdsupra.com/legalnews/doj-shifts-position-on-web-access-56084/
> 
> Specifically, note that “at issue are not videos that students enrolled in paid or free courses may be required to view. Instead, at issue are videos of general interest (e.g., a lecture posted by a professor, a speech given at the university, or even a campus organization’s event).”
> 
> From my reading of this, it pretty much summarily dismisses any attempt to indemnify the university by prioritizing content based on a tiered system – all content should be accessible, even casual content of a non-educational nature.  DOJ has stated many times that the obligation to make university content accessible is pre-existing, so this is not a different interpretation of 504 so much as a clarification.  Essentially “we’ve given you time to work this stuff out but at this point most of your web content has been recoded several times  over the past decade and that’s long enough to have made any needed accessibility changes”.
> 
> I’m not even worried so much about standard web sites at this point because what few accessibility issues we have come across are minimal – while they may not meet the strict letter of the law in terms of accessibility, as far as I know, nobody has been sued for having an improperly tagged language attribute on a page.  The shift has been toward online course content and there is no doubt that if universities are teaching online courses, that material needs to be accessible out of the box, including the captioning of video content.

**Comment by loganfranken** [comment]
I think there is a general agreement (and especially with Mark's comments above) that, if we keep the page, the language at the top of the Tiers page will need to be rewritten to target the specific set of standards to which the Tiers system applies.

To that end, those who are in support of keeping the page may want to come to the next WSG meeting with some suggestions for how to rewrite that text?

Here it is again (from the [Tiers page](http://webguide.ucsb.edu/tiers/)):

(I've bolded the problematic line)

> The Web Standards Group has grouped UCSB websites into three tiers, based on various factors: association with the University identity, sphere of influence, frequency and volume of use, expected lifetime, and other factors. **A website's position reflects how strongly we believe that site should adhere to the standards defined in this guide.** All campus units, regardless of the tier, should make every effort to comply with WSG accessibility recommendations. The tiers are described below, followed by examples of selected sites in each tier. Other sites will be added in the future.

**Comment by loganfranken** [comment]
Sorry I didn't make it to the meeting!

Where did we land with this?

**Comment by loganfranken** [comment]
Ahh, sorry: I see the meeting was cancelled.

**Comment by tenken** [comment]
I have taken a stab at rewritting the Tiers page. I have not yet tried to rewrite the Graphics Identity Page ... but I'll try to get to that shortly:

https://github.com/tenken/ucsb-wsg.github.io/blob/update-tiers-recommendations/tiers.md

**Comment by alexparraga** [comment]
Thanks, David. Our office will be providing updated and more comprehensive graphics identity content within a month, as previously discussed. So don't worry about that one. We can discuss whether it should still live on WSG or link out to a separate page on the ucsb.eduhttp://ucsb.edu domain.

**Comment by loganfranken** [comment]
During the meeting just now, we decided on the following:
- Remove the "Three Tiers" page
- Add 1-2 paragraph(s) to the home page that includes the following:
  - Language that reemphasizes that WSG is not an enforcement body, every site _should strive_ to meet the standards, but may be limited by constraints (resources, technical, etc.)
  - Link to this GitHub repository where people can interact with the community surrounding the guide

Please let me know if I got any of this wrong!

**Comment by mer-ucsb** [comment]
Hi Logan

Through medical appointments, I had to miss this meeting. I am seriously
concerned that the three tiers page is being removed.

Did anyone speak to how helpful and supportive this three tier page is when
trying to get the non-technical website decision makers to _understand_
what look-and-feel the whole of the ucsb-web ux experience should be???

Helping the web programmers/designers is precisely why this three tier page
was created in the first place. People seem to forget this and think that
seamless ux experience unimportant.

I feel like I am just beating a dead horse.

Margaret

**Comment by tenken** [comment]
There were many points of views and interpretations regarding the Three Tiers; at one point in the meeting Logan listed 3-4 viewpoints expressed in the room. Including what you're voicing regarding ucsb UX.

But the bottom line is nothing is enforceable from the webguide, and using the word _Tiered_ in the webguide seems to advocate that websites may _possibly_ not adhere to the best practices referred to in the webguide -- we no longer want such ambiguity in the webguide as to what standards as followed by certain tier(s) of websites. This is voiced by Alex and Mark Grosch in the above thread.

To that end the WSG is seeking to clarify that:
1. we're simply advocating that all websites **must** be accessible and follow all best practices where possible as per UC Policy. The webguide, WSG members, and Github (here) exist as a _usable_ resources for campus entities to learn from our experience on how best to implement projects given  constraints such as: developer experience, time, financial resources, etc.
2. With regards to Branding and Graphic Identity new resource(s) will soon be available and referenced on the webguide for voluntary campus adoption.

Individuals with questions are free to contact the WSG group through 1 of the many existing avenues of communication.

**Comment by mer-ucsb** [comment]
Hi David

With regards to ... Branding and Graphic Identity new resource(s) will soon
be available and referenced on the webguide for voluntary campus adoption
...
THIS is the part that I was talking about. The importance of the
main/highly trafficked UCSB sites to all have a reasonable seamless look.
It is this point that remains important and highly relevant and is/was part
of the three tier page.

Margaret

**Comment by tenken** [comment]
I have made a 2nd attempt at revising the Tiers content:
- remove Tiers page.
- add notice to graphics identity page announcing upcoming changes.
- add history information of move to Github, and mention the Issues (here) as a resource for _any_ to contact the WSG with inquiries.

https://github.com/ucsb-wsg/ucsb-wsg.github.io/compare/master...tenken:master

Any feedback is appreciated.

**Comment by UCEAP-JCOLON** [comment]
Excellent.
Thanks David!

**Comment by loganfranken** [comment]
Hey @tenken! The updates look good to me!

I thought we had discussed also adding a small paragraph to the home page clarifying that we _do not_ enforce anything?

**Comment by tenken** [comment]
@Logan we did discuss that. But the idea that we foster best practices is covered in the **Mission Statement**.

We state on the about page:

> The WSG does not police campus websites.

So, I was trying to avoid being repetitive throughout the document.

**Comment by UCEAP-JCOLON** [comment]
Were we going to add something to the effect that the recommendations here in are best practices that we hope campus websites strive to achieve, yada yada. This allowing for the separation of the tiers as it is a recommendation that all sites strive to achieve those practices...

**Comment by tenken** [comment]
@UCEAP-JCOLON I wasn't explicitly going to bring in that back history to any part of a page and simply let the Tiers concept die off. The guide in numerous places already states we're hoping projects strive to meet the high-standards set forth.

I'm trying not to rewrite everything (myself) as my writing style is pretty different from the current "voice" of the document.

**Comment by loganfranken** [comment]
With that, I'm happy with the rewrite: my vote is to move forward with the PR.
